### DISCUSSION TITLE
Can we consider a different default indentation?

### DISCUSSION THREAD

**Discussion by toastal** [tweag/nickel#1745] [Ideas]
I’ve been happy with `topiary` and disabling the automatic expandtab in [Vim](https://github.com/nickel-lang/vim-nickel/issues/8), but I see now a new built in called `nickel format` which uses default settings. As I mentioned in Vim plugin issue, I find these defaults difficult to work with. Before things get too solidified, is the community opposed to exploring a default indentation with spacing >2 or tabs?

For now, I’m still happy with what what `topiary`’s settings can provide me, but I’m just working with myself when it comes to Nickel code so I have the flexibility to do what I want, but having worked in teams, sometimes you can’t get much of a say with everyone pointing to a tool’s defaults.

**Comment by vkleen** [comment]
`nickel format` is just `topiary` under the hood, so you definitely wouldn't loose anything (except for ease of installation) by just staying with `topiary`. Just to clarify, are you using a custom query file for `topiary` or is there an option to use tabs instead of spaces there? Offering the same CLI options for `nickel format` that `topiary` offers seems like a very reasonable thing to do.

**Reply by toastal** [reply]
In the same directory topiary & nickel format had different outputs. However, I do have a `.topiary/languages.toml` file that overrides `indent`.

**Reply by toastal** [reply]
```sh-session
$ nickel --version
nickel-lang-cli nickel 1.3.0 (rev cargore)
$ topiary --version
topiary-cli 0.3.0
```
